### DISCUSSION TITLE
Supported needed : Adding Assertion for test step

### DISCUSSION THREAD

**Discussion by ManimaranChadrababu** [Ginger-Automation/Ginger#2004] [Q&A] [answered]
Hello Team, 
I am trying out ginner framework for test automation for my project.
I can able to create and execute test but all we need is to validate the step with assertion ,
any idea or support is appreciated.

Thanks in Advance
-Mani

**Comment by JinendraGhodke** [comment, marked as answer]
In the output values, you can use various operators and conditions to perform the assertion.
Please refer to below help document for more details 

https://ginger-automation.github.io/Ginger-Web-Help/assets/Ginger_By_Amdocs/Business_Flows/Actions/Action_Generic_Configurations/Action_Output_Values.htm?rhsearch=output%20values&rhhlterm=output%20%E2%80%98output%20values%20values%E2%80%99%20value
